jfrog_platform_self-hosted/README.md

```diff
@@ -0,0 +1,274 @@
+## Overview
+
+[JFrog][1] is a universal, hybrid, and end-to-end DevOps platform. This integration helps any JFrog self-hosted customer seamlessly stream logs, violations and metrics from JFrog Artifactory and JFrog Xray straight into your Datadog tool. This integration comes ready with in-built support for Datadog [log pipelines][2] which enriches and indexes the logs and makes them more searchable and treatable using Datadog [facets][3].  
+
+Let JFrog know how we can improve the integration. Feel free to visit our GitHub for more detailed documentation at: [github.com/jfrog/log-analytics-datadog][4].
+
+### **IMPORTANT NOTE : The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will be deprecated by July 31st 2023.** 
```

Comment:
Instead of placing the deprecation note here, recommend using an alert at the top of the page under Overview section above.
```suggestion
<div class="alert alert-warning">The JFrog Platform (self-hosted) check replaces the <a href="https://docs.datadoghq.com/integrations/jfrog_platform/">JFrog Platform check</a> which is deprecated. JFrog metrics are gathered by FluentD.</div>
```